resources/views/activities/index.blade.php

```diff
@@ -0,0 +1,107 @@
+@extends('layouts.app')
+
+@section('title', __('main.users'))
+
+@section('content')
+
+    <!-- Alerts - OPEN -->
+    @include('parts.alerts')
+    <!-- Alerts - CLOSE -->
+
+    <!-- Language for dates - OPEN -->
+    @php
+        \Date::setLocale(Session::get('locale'));
+    @endphp
+    <!-- Language for dates - CLOSE -->
+
+    <!-- Content - OPEN -->
+    <div class="container margin-top padding-bottom">
+
+        <div class="row">
+            <div class="col-sm"></div>
+            <div class="col-sm">
+                <h3 class="margin-bottom-sm text-center">
+                    {{ __('main.activity_log') }}
+                </h3>
+            </div>
+            <div class="col-sm text-right">
+            <a href="{{ route('activities_delete_all') }}">     <button type="button" class="btn btn-danger"> <span style="padding-right:0.5em;" class="octicon octicon-trashcan"></span> Clear Log</button></a>
```

Comment:
Not indented correctly. Also doesn't make sense a <button> inside a <a>. It should be replaced with <a href="" class="btn btn-danger> as this works as a button.

Reply:
Indentation added. If anchor tag (a) is added directly with the bootstrap classes then we have to modify a bit the styles of anchor tag. Hence when using button in layout, we can leverage the bootstrap classes without any modificiation.